fixtures/robe/robin-pointe.json

```
@@ -0,0 +1,2157 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Robin Pointe",
```

```suggestion
  "name": "Pointe",
```

Website: https://www.robe.cz/pointe